src/types/index.mts

```diff
@@ -11,5 +12,7 @@ export interface Answer {
   packageTemplate: string;
   namespacedDir: string;
   packageOwnerTeamId: string;
+  packageUsesDataDog: true;
```

Comment:
Should this be `boolean`?

Reply:
Yep! Nice catch!

Reply:
Even TS can't stop me from being dumb! 🤣

Reply:
Honestly, I'm surprised it didn't yell at you here. `true` is a value, not a type, right?